spec.bs

```diff
@@ -3834,36 +3891,49 @@ from querying the server during an auction.
-
-      * "NameReport"
-      * the [=serialization of an origin|serialization=] of |ig|'s [=interest group/owner=]
-      * the [=URL serializer|serialization=] of |ig|'s [=interest group/bidding url=]
-      * the [=URL serializer|serialization=] of |igAd|'s [=interest group ad/render url=]
-      * If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-          [=map/exists=]:
-         * "BuyerAndSellerReportingId"
-         * |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-      * Otherwise, if |igAd|'s [=interest group ad/buyer reporting ID=]
-          [=map/exists=]:
-         * "BuyerReportingId"
-         * |igAd|'s [=interest group ad/buyer reporting ID=]
-      * Otherwise:
-         * "IgName"
-         * |ig|'s [=interest group/name=].
-   1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
+  [=interest group ad=] |igAd|, and a [=string=]-or-null |selectedReportingId|:
+    1. Let |middle| be the [=string/concatenation=] of the following strings separated with U+000A (LF):
+      1. the [=serialization of an origin|serialization=] of |ig|'s [=interest group/owner=]
+      1. the [=URL serializer|serialization=] of |ig|'s [=interest group/bidding url=]
+      1. the [=URL serializer|serialization=] of |igAd|'s [=interest group ad/render url=]
+    1. Let |keyString| be an empty [=string=].
+    1. If |selectedReportingId| is not null:
+      1. Set |keyString| to be the [=string/concatenation=] of the following strings separated with U+000A (LF):
+        1. "SelectedBuyerAndSellerReportId"
+        1. |middle|
+        1. The result of [=compute the key part for one of multiple reporting ids=] given |selectedReportingId|
+        1. The result of [=compute the key part for one of multiple reporting ids=] given |igAd|'s 
+          [=interest group ad/buyer and seller reporting ID=]
+        1. The result of [=compute the key part for one of multiple reporting ids=] given |igAd|'s
+          [=interest group ad/buyer reporting ID=]
+    1. Otherwise: 
+      1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=] [=map/exists=]:
+        1. Set |keyString| to be the [=string/concatenation=] of the following strings separated with U+000A (LF):
+          1. "BuyerAndSellerReportId"
+          1. |middle|
+          1. |igAd|'s [=interest group ad/buyer and seller reporting ID=]
+      1. Otherwise if |igAd|'s [=interest group ad/buyer reporting ID=] [=map/exists=]:
+        1. Set |keyString| to be the [=string/concatenation=] of the following strings separated with U+000A (LF):
+          1. "BuyerReportId"
+          1. |middle|
+          1. |igAd|'s [=interest group ad/buyer reporting ID=]
+      1. Otherwise:
+        1. Set |keyString| to be the [=string/concatenation=] of the following strings separated with U+000A (LF):
+          1. "IgName"
+          1. |middle|

```

Comment:
We need a `U+000A (LF)` between `|middle|` and `interest group update`.

Reply:
As we discussed this is covered by "separated with U+000A (LF)".